flang/lib/Lower/OpenMP/Utils.cpp

```diff
@@ -87,6 +88,27 @@ mlir::Type getLoopVarType(Fortran::lower::AbstractConverter &converter,
   return converter.getFirOpBuilder().getIntegerType(loopVarTypeSize);
 }
 
+Fortran::semantics::Symbol *
+getIterationVariableSymbol(const Fortran::lower::pft::Evaluation &eval) {
```

Comment:
Nit: Since it's not really OpenMP-specific, maybe flang/include/flang/Lower/Support/Utils.h would be a better place to put it, though that would probably require adding the corresponding .cpp file (that header currently only has inline definitions). I'll leave it up to you to do that, find a more suitable spot or just leave it where it is.

Reply:
We can consider this for another PR.